sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -393,6 +393,45 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
    * This function goes through the input once to determine the input schema. To avoid going
    * through the entire data once, specify the schema explicitly using [[schema]].
    *
+   * You can set the following CSV-specific options to deal with CSV files:
+   * <li>`sep` or `delimiter` (default `,`): sets the single character as a delimiter for each
+   * field and value.</li>
+   * <li>`encoding` or `charset` (default `UTF-8`): decodes the CSV files by the given encoding
```

Comment:
Sure. Thank you. Let me correct them.
Also, I will add a document as soon as R csv API is added. 
I tried this in https://github.com/apache/spark/pull/11457 but had to ask someone to do this as I am not really familiar with R.